SR2019/2019-03-27-final-competition-details.md

```diff
@@ -0,0 +1,25 @@
+---
+to: Student Robotics 2019
+subject: Final competition details
+---
+
+Hi all,
+
+The competition is now less than 2 weeks away! We have some additional important information for you.
+
+Most importantly is media consent. Media consent forms must be sent to us by 5pm Thursday 4th April. After then most of our volunteers will be on site and not handling
+forms. If you've not returned a media consent form by then, it will take significantly longer to pass through reception, and some competitors may be turned away.
+
+You can download the media consent form from [our website](https://studentrobotics.org/events/sr2019/competition/).
+
+Parents and other visitors do not require tickets, but are required to sign media consent forms. These forms should be bought with them, or can be signed at reception.
+
+Please ensure that your team bring at least 1 laptop or appropriate device with them to the competition to access the docs and IDE on. We are unable to provide any computing devices for competitor use at the competition. We will be providing a wireless internet connection to all teams.
+
+Any tools that your team requires to modify their robot must be brought to the competition by the team: we cannot provide any. In particular, the use of any dangerous tools must be supervised by the responsible adult for the team.
+
+All robots are subject to a safety check before the start of the competition (see appendix B in the rulebook). We may disqualifying any dangerous robots, so please ensure the team have checked their robot against the safety checklist.
```

Comment:
> We may disqualifying any dangerous robots

Missing word here? I'd also upgrade "may" to something stronger -- perhaps "Robots which do not pass safety checks will not be allowed in the arena"?